packages/cli/test/cli.test.js

```diff
@@ -123,4 +123,17 @@ describe("CLI", () => {
     },
     timeout
   );
+
+  test(
+    "should successfully slice a test image to IIIF",
```

Comment:
This test description and the assertion do not match; the description is that an image is "successfully sliced" though the assertion is that a directory exists. Alone the assertion is not very useful however to correct it without writing additional tests update the description to something like `test("directory is created for iiif image tiles", async (done) => {`